src/main/java/com/softserveinc/ita/pageobjects/CheckoutPage.java

```diff
@@ -59,18 +62,25 @@ public int getProductPrice() {
         return parseInt(productPrice);
     }
 
-    public int getProductAmountPrice() {
-        var productAmountPrice = $(".js-product-amount dd")
+    public int getProductTotalPrice() {
+        var productTotalPrice = $(".js-product-amount dd")
                 .getText()
                 .replaceAll(" ₴", "");
 
-        return parseInt(productAmountPrice);
+        return parseInt(productTotalPrice);
     }
 
     public int getProductQuantity() {
-        var productQuantity = $(".js-product-quantity dd")
-                .getText();
+        var productQuantity = $(".js-product-quantity dd").getText();
 
         return parseInt(productQuantity);
     }
+
+    public boolean didCheckoutPageOpenCorrectly() {
```

Comment:
it will always return true, what purpose of this method?

Reply:
![image](https://user-images.githubusercontent.com/34237412/143203167-169d34d2-afd1-4af7-8200-92d637576dbd.png)

Reply:
rename to refreshIfErrorExists()

move to base page

return statement generic

you can try do it through while

var count = 0;

while($("rz-critical-error.ng-star-inserted").exists() && count < 5){
refresh();
}

Reply:
i will probably ignore this problem because i can't reproduce this error and can't test if this method works or not, so im just wasting my time on this. Probably better to say that test was 'Blocked' if this error shows because it's Rozetka's error.

Reply:
I got it